scout/models/case/case_loading_models.py

```diff
@@ -0,0 +1,509 @@
+
+    return real_folder_images
+
+
+class RawCustomImages(BaseModel):
+    """This class makes a preliminary check that custom_images in the load config file has the expected structure for the custom images.."""
+
+    str_variants_images: Optional[List[Image]] = []
+    case_images: Optional[Dict[str, List[Image]]] = {}
+
+    @model_validator(mode="before")
+    def set_custom_image(cls, values: Dict) -> "RawCustomImages":
+        values["str_variants_images"] = values.get(
+            "str_variants_images",
+            values.get("str"),
+        )
+        values["case_images"] = values.get(
+            "case_images",
+            values.get("case"),
+        )
+        return values
+
+
+class ParsedCustomImages(BaseModel):
+    """This class corresponds to the parsed fields of the custom_images config item."""
+
+    str_variants_images: Optional[List[Image]] = []
+    case_images: Optional[Dict[str, List[Image]]] = {}
+
+
+#### Case - related pydantic models ####
+
+
+class CaseLoader(BaseModel):
+    analysis_date: Optional[Any] = datetime.now()
+    assignee: Optional[str] = None
+    case_id: str = Field(alias="family")
+    cnv_report: Optional[str] = None
+    cohorts: Optional[List[str]] = None
+    collaborators: Optional[List[str]] = None
+    coverage_qc_report: Optional[str] = None
+    custom_images: Optional[Union[RawCustomImages, ParsedCustomImages]] = None
+    default_panels: Optional[List[str]] = Field([], alias="default_gene_panels")
+    delivery_report: Optional[str] = None
+    display_name: Optional[str] = Field(alias="family_name")
+    exe_ver: Optional[str] = None
+    family: Optional[str] = None
+    gene_fusion_report: Optional[str] = None
+    gene_fusion_report_research: Optional[str] = None
+    gene_panels: Optional[List[str]] = []
+    genome_build: Union[str, int] = Field(38, alias="human_genome_build")
+    individuals: Union[List[SampleLoader]] = Field([], alias="samples")
+    lims_id: Optional[str] = None
+    madeline_info: Optional[str] = Field(None, alias="madeline")
+    multiqc: Optional[str] = None
+    multiqc_rna: Optional[str] = None
+    owner: Optional[str] = None
+    peddy_ped: Optional[str] = None  # Soon to be deprecated
+    peddy_ped_check: Optional[str] = Field(None, alias="peddy_check")  # Soon to be deprecated
+    peddy_sex_check: Optional[str] = Field(None, alias="peddy_sex")  # Soon to be deprecated
+    phenotype_terms: Optional[List[str]] = None
+    rank_model_version: Optional[str] = None
+    rank_score_threshold: Optional[int] = 0
+    reference_info: Optional[str] = None
+    RNAfusion_inspector: Optional[str] = None
+    RNAfusion_inspector_research: Optional[str] = None
+    RNAfusion_report: Optional[str] = None
+    RNAfusion_report_research: Optional[str] = None
+    smn_tsv: Optional[str] = None
+    sv_rank_model_version: Optional[str] = None
+    synopsis: Optional[Union[List[str], str]] = None
```

Comment:
This syntax is kind of weird! The List[str] can be there until we validate, then it is always str. Or is it for legacy entries?

Reply:
Well this is what's accepted in the config file (None, str or list of strings). not what the synopsis will look like after?